Added general styling and placeholder images for About Us page.

andimiya opened the pull request:
Could use a little bit more styling fanciness.
Closes #9

andimiya committed:
Added about us content and styled it like the homepage

andimiya committed:
Added ijmages

andimiya committed:
Styling to wrap images when screen size is small

brockinit commented on client/src/containers/AboutUsContainer/index.js:
```diff
@@ -1,12 +1,79 @@
+        </div>
+        <div style={styles.aboutBody}>
+          <h2 style={styles.h2}>Brock</h2>
+          <p>A self-proclaimed data nerd with a distaste for baseless arguments and emotional decisions, Brock has always been about bringing facts to the gridiron. When he isn't serving up solutions at Sudokrew, a software shop in Hawaii, he's neck deep in deep learning and developing ways in which he can predict seemingly randoms events, such as sporting events and stock prices. If forced to na ...
```
Let's add this information into a file called `aboutInfo.json` and import it into the `AboutUs` container. We can then run `.map` during component rendering and allow the content to be easily swapped out rather than hardcoded into the markup.

brockinit committed:
Change AboutUs to a stateless component

andimiya committed:
🔥 Componentized About Us page

Restructured About Us page into components that can be shared across different pages" #9

andimiya committed:
🎨 Added title components to homepage

brockinit commented:
Looking better!
